conf/modules.config

```diff
@@ -126,9 +126,10 @@ process {
         ]
     }
 
-    withName: BBMAP_ALIGN {
+    withName: '.*:BBMAP_ALIGN' {
+        ext.args = "trimreaddescriptions=t pigz=t"
         publishDir = [
-            [
+            [   
```

Comment:
You have trailing whitespace (i.e. space characters at the end of the line) here, not allowed in nf-core.